datafusion/core/tests/sql/information_schema.rs

```diff
@@ -669,98 +669,3 @@ async fn show_external_create_table() {
 async fn plan_and_collect(ctx: &SessionContext, sql: &str) -> Result<Vec<RecordBatch>> {
     ctx.sql(sql).await?.collect().await
 }
-
-#[tokio::test]
-async fn show_variable_in_config_options() {
-    let ctx =
-        SessionContext::with_config(SessionConfig::new().with_information_schema(true));
-    let sql = "SHOW datafusion.execution.batch_size";
-    let results = plan_and_collect(&ctx, sql).await.unwrap();
-
-    let expected = vec![
-        "+---------------------------------+---------+",
-        "| name                            | setting |",
-        "+---------------------------------+---------+",
-        "| datafusion.execution.batch_size | 8192    |",
-        "+---------------------------------+---------+",
-    ];
-
-    assert_batches_eq!(expected, &results);
-}
-
-#[tokio::test]
-async fn show_all() {
```

Comment:
moved to a data file rather than .rs file 🎉

Reply:
Yeah, step by step migration in progress